frontend/src/app/game/pages/outro/outro.component.html

```diff
@@ -11,7 +11,7 @@ <h1>Vaše hra v roli vlády skončila</h1>
     </div>
   </div>
 
-  <cvd-scatter-graph
+  <cvd-scatter-graph *ngIf="(datasets$ | async)"
```

Comment:
You can do smth like this: `*ngIf="(datasets$ | async) as datasets"` and on row 16. [datasets]="datasets"